TrackMySleepQuality-Starter/app/build.gradle

```diff
@@ -71,5 +75,10 @@ dependencies {
     // Navigation
     implementation "android.arch.navigation:navigation-fragment-ktx:$navigationVersion"
     implementation "android.arch.navigation:navigation-ui-ktx:$navigationVersion"
+
+    // Testing
```

Comment:
Were these testing dependencies recommended by Android Studio?

Reply:
In room and testing codelab these imports are used.